### PR TITLE
[KKT] add two folders Sparse and Dense for KKT formulations

### PR DESCRIPTION
**Pull request by frapac**
cc @amontoison

### CODE REVIEW

File: src/KKT/Dense/utils.jl [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    # / x
+    mul!(yx, kkt.jac', x)
+    # / s
+    ys .= -@view(x[ind_ineq])
+    return
+end
+
+function compress_jacobian!(kkt::AbstractDenseKKTSystem)
+    return
+end
```
Comment (amontoison) [inline src/KKT/Dense/utils.jl:27]: We already have `compress_hessian!(kkt::AbstractKKTSystem) = nothing` in `src/KKT/KKTsystem.jl`.

**Review by amontoison — APPROVED** [review]

**Comment by amontoison** [comment]
Thanks @frapac!

**Comment by amontoison** [comment]
Good to be merged?

**Comment by frapac** [comment]
Merging this PR as this is only cosmetic changes.

**Pull request merged**